docs/pages/versions/unversioned/sdk/application.md

```diff
@@ -0,0 +1,141 @@
+```
+
+### `Application.getInstallReferrerAsync()`
+
+**Android only.** Gets the referrer URL of the installed app with the [`Install Referrer API`](https://developer.android.com/google/play/installreferrer) from the Google Play Store.
+
+#### Returns
+
+A `Promise` that resolves to a `string` of the referrer URL of the installed app.
+
+**Examples**
+
+```js
+await Application.getInstallReferrerAsync();
+// "utm_source=google-play&utm_medium=organic"
+```
+
+### `Application.getInstallationTimeAsync()`
+
+Gets the time the app was first installed onto the device. The time would change to the reinstalled date if uninstall and reinstall the app.
+
+On iOS, it depends on the [`NSFileCreationDate`](https://developer.apple.com/documentation/foundation/nsfilecreationdate?language=occ) of the app's document root directory. On Android, it depends on the [`PackageInfo.firstInstallTime`](https://developer.android.com/reference/android/content/pm/PackageInfo.html#firstInstallTime).
+
+#### Returns
+
+Returns a `Promise` that resolves to a `Date` object that represents the first time the app was installed on the device.
+
+**Examples**
+
+```js
+await Application.getFirstInstallTimeAsync();
+// 2019-07-18T18:08:26.121Z
+```
+
+### `Application.getLastUpdateTimeAsync()`
+
+**Android only.** Gets the last time the app was updated from the Google Play Store.
+
+#### Returns
+
+Returns a `Promise` that resolves to a `Date` object that represents the last time the app was updated via the Google Play Store (Android).
+
+**Examples**
+
+```js
+await Application.getLastUpdateTimeAsync();
+// 2019-07-18T21:20:16.887Z
+```
+
+## Error Codes
+
+| Code                                              | Description                                                                                                                                                                                                                                                                                                                                                                                   |
+| ------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ERR_APPLICATION_PACKAGE_NAME_NOT_FOUND            | Error code thrown for `firstInstallTimeAsync` and `lastUpdateTimeAsync`. This may be thrown if the package information or package name could not be retrieved.                                                                                                                                                                                                                                |
```

Comment:
```suggestion
| ERR_APPLICATION_PACKAGE_NAME_NOT_FOUND            | Error code thrown by `getInstallationTimeAsync ` and `getLastUpdateTimeAsync`. This may be thrown if the package information or package name could not be retrieved.                                                                                                                                                                                                                                |
```